example/index.js

```diff
@@ -13,21 +13,97 @@ div.style.bottom = 0;
 const map = new mapboxgl.Map({
   container: div,
   style: 'mapbox://styles/mapbox/streets-v9',
-  center: [-79.387, 43.656],
-  zoom: 13.5
+  center: [-93.87, 41.65],
+  zoom: 3
 });
 
 map.on('load', () => {
+  map.addSource('data', {
```

Comment:
Can this be split into its own example instead? Demonstrating adding a custom source and applying accessibility to it.